.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
Why do we need this file to resolve the missing link?

Reply:
no need to change that file. Removed from the pr